format.go

```diff
@@ -41,6 +41,16 @@ func DetectFormat(name string) Format {
 	return (*noFormat)(nil)
 }
 
+// FormatExt returns file extension for a format
+func FormatExt(format Format) string {
```

Comment:
same, does it need to be public? always remember: once something is public, you cannot remove it without deprecating it, major release changes, etc